core/templates/dev/head/domain/exploration/SolutionObjectFactory.js

```diff
@@ -68,8 +68,7 @@ oppia.factory('SolutionObjectFactory', [
       var explanation = (
         $filter('convertToPlainText')(this.explanation));
       return (
-        '[' + solutionType + ' solution is ' + correctAnswer + '] ' +
-        explanation);
+        solutionType + ' solution is ' + correctAnswer + '. ' + explanation);
```

Comment:
Some interactions already have quotation marks actually. That's why I used braces.